src/chara.js

```diff
@@ -393,10 +393,8 @@ var Chara = CreateClass({
     handleSelectEvent: function (key, e) {
         var newState = this.state;
 
-        if (key == "isConsideredInAverage") {
-            newState[key] = (newState[key] == false) ? true : false
-        } else if (key == "EXLBWED") {
-            newState[key] = (newState[key] == false) ? true : false
+        if (key == "isConsideredInAverage" || key == "EXLBWED") {
+            newState[key] = (newState[key] == false)
```

Comment:
I do not know what do this, just javascript expression
- boolean context can be `!newState[key]` ?
  - although when newState[key] was 0, `!0` also true. then if condition can check the type
    - if (typeof newState[key] === 'boolean') { // key: isConsideredInAverage, EXLBWED cases
- switch (key) is another choice. if another type of key is add in future.

```jsx
switch (key) {
    case "isConsideredInAverage":
    case "EXLB":
        newState[key] = (newState[key] == false);
        break;
    default:
        newState[key] = e.target.value;
        break;
}
```

for now, it's only two keys then the current code is ok.
- [ ] just remember **";" at the end.**

Reply:
BTW, why inverse flag here?

Reply:
if e.target was Checkbox and e.target.checked,

switch logic per e.target then
when another key is add, no need to change here.

Reply:
Updated to check if it is checkbox instead of keys on 1185c63